### PR TITLE
PMM-4309 submodules cleanup

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -369,6 +369,7 @@ pipeline {
                                         echo "export PATH=$PWD/pmm2-client-2.0.0/bin:$PATH" >> ~/.bash_profile
                                         source ~/.bash_profile
                                         pmm-admin --version
+                                        cat pmm-agent.log
```
Comment (AlekSi) [inline pmm/aws-staging-start.groovy]: This file does not exist at this point.

Comment (puneet0191) [reply pmm/aws-staging-start.groovy]: When installation is done via tar or in all cases?

Comment (AlekSi) [reply pmm/aws-staging-start.groovy]: I mean, there is nothing before that line that creates this file. pmm-agent itself does not creates it, it logs only to stderr.

Comment (puneet0191) [reply pmm/aws-staging-start.groovy]: oh yes, I realize what I did there, pushing fix.

**Pull request merged**